store.go

```diff
@@ -5,17 +5,11 @@ import (
 	"strings"
 
 	"github.com/docker/libkv/store/boltdb"
-	"github.com/docker/libkv/store/consul"
-	"github.com/docker/libkv/store/etcd"
-	"github.com/docker/libkv/store/zookeeper"
 	"github.com/docker/libnetwork/datastore"
 	"github.com/sirupsen/logrus"
 )
 
 func registerKVStores() {
-	consul.Register()
-	zookeeper.Register()
-	etcd.Register()
 	boltdb.Register()
```

Comment:
So, looking at these; I _think_ boltdb is still used, but I'm not sure; wondering if @arkodg knows more about what of this we can all remove

Reply:
ya AFAIK we use boltdb for local drivers such as bridge